app/views/kata/_file_new_rename_delete.html.erb

```diff
@@ -83,6 +83,7 @@ $(function() {
     var scrollTop = oldFile.scrollTop();
     var scrollLeft = oldFile.scrollLeft();
     var caretPos = oldFile.caret();
+
     $(oldFile).closest('tr').remove();
```

Comment:
If you think it should be that then I trust you :-)
Feel free to just go ahead. No need to issue a pull request.